### PR TITLE
bill-it payment sdk

### LINKED ISSUE #31: `Payments` methods

### PR DESCRIPTION
**Pull request by phemmylintry**
closes #31

### CODE REVIEW

**Comment by phemmylintry** [comment]
@akalipetis @parisk Please review. Thanks

File: billit/client.py [hunk 1]
```diff
@@ -678,3 +679,43 @@ def create(
...
+    def create(
+        self,
+        customer_id: int,
+        date_occurred: str,
+        amount: int,
+        payment_method: int,
+        payment_type: int,
+        amount_left_over: int,
+        selections_amount: int,
+        invoices_paid: List,
```
Comment (parisk) [inline billit/client.py:708]: @phemmylintry let's create an `InvoicePaid` type according to the payload described in https://developer.billit.io/docs/billitdocs/e8bc28eaa24ea-create-a-new-payment and mark `invoices_paid` as a list of `InvoicePaid` (it should be `InvoicePaid[]`).

**Review by parisk — CHANGES_REQUESTED** [review]
Great job @phemmylintry! Just a comment.

File: billit/client.py (hunk 1 above)
Comment (phemmylintry) [reply billit/client.py:708]: @parisk I'm not sure I get it. The type is a list, right?

Comment (parisk) [reply billit/client.py:708]: Yes. `List` is quite generic. We want to indicate that it is a list of `PaidInvoice`. So we need to:

- [ ] Declare a new `PaidInvoice` type, similarly to `Tax`[^1]
- [ ] Indicate that `invoices_paid` is a list of `PaidInvoice` (this should be enough `: PaidInvoice[]`)

Let me know if you would like further clarifications

[^1]: https://github.com/withlogicco/billit-python-sdk/blob/a5e028185f80d73accbeab55ad35d8d0fc27b160/billit/utils/tax_utils.py#L4-L7

**Review by parisk — APPROVED** [review]
LGTM 🍰

**Pull request merged**